src/rotation.rs

```diff
@@ -25,7 +25,8 @@ use serde;
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "serde", serde(bound(serialize = "T: serde::Serialize", deserialize = "T: serde::Deserialize<'de>")))]
 pub struct Rotation2D<T, Src, Dst> {
-    pub angle : T,
+    /// Angle in radians, counter counterclockwise
```

Comment:
Please remove the indication of whether the orientation is clockwise or counter-clockwise. It depends on what users of euclid chose the orientation of the y axis to mean (whether it means up or down).